indico/modules/events/editing/templates/emails/comment_notification.txt

```diff
@@ -1,11 +1,19 @@
 {% extends 'events/editing/emails/base_email.txt' %}
 
-{% block subject -%}{% trans %}New comment{% endtrans %}{%- endblock %}
+{% block subject -%}
+{% if programme_code -%}
+    {% trans %} New comment on contribution {{contribution_id}} with programme code {{programme_code}}{% endtrans %}
+{%- else -%}
+    {% trans %} New comment on contribution {{contribution}}{% endtrans %}
+{%- endif %}
+{%- endblock %}
 
 {% block body_text -%}
 {% if author_name -%}
-    {% trans name=author_name %}{{ name }} has posted a new comment.{% endtrans %}
+    {% trans %}{{ author_name }} has posted a new comment on contribution {{contribution_id}}:{% endtrans %}
```

Comment:
I'd also check here if theres a programme code. Maybe have something at the beginning like:
```jinja
{% set title=contribution.code or contribution.title %}
```
Or maybe even only pass `title=contribution.code or contribution.title` to the template, if you don't need the contribution's code or title anywhere else.

Also, you need to pass any params to i18n (the `trans` macro) in order for them to be properly translated (like how the name was being passed before)

Reply:
Plain attributes like `{{ author_name }}` can be used directly - the only reason to put it in the `{% trans ... %}` is if you want the placeholder in the extracted string to have a different name.